src/app/lib/utilities/getEmbedUrl/index.js

```diff
@@ -0,0 +1,31 @@
+import isLive from '../isLive';
+
+const AV_ROUTE = 'ws/av-embeds';
+
+const LIVE_URL = 'https://polling.bbc.co.uk';
+const TEST_URL = 'https://polling.test.bbc.co.uk';
+
+const shouldRenderLiveData = queryString => {
+  return (
+    !isLive() &&
```

Comment:
I feel this logic would be confusing to anyone looking at this file without context - returning `false` from the `shouldRenderLiveData()` function if we're live.

I think it would be more accurate to return `true` from this function if we **are** live.

Would also allow lines 17-19 to be delete